assets/section-image-banner.css

```diff
@@ -98,14 +135,34 @@
   display: flex;
   position: relative;
   width: 100%;
+  align-items: center;
   justify-content: center;
+  z-index: 2;
+}
+
+@media screen and (max-width: 749px) {
+  .banner--mobile-bottom:not(.banner--stacked) .banner__content {
+    order: 2;
+  }
 }
 
 @media screen and (min-width: 750px) {
   .banner__content {
     padding-bottom: 5rem;
     padding-top: 5rem;
```

Comment:
Now I see why I couldn't reproduce it... I never thought of using a logo as a background image.

Reply:
Yeah, I spent so much time on this one, but in the end, I feel like the section is working exactly as expected. If we force a taller height, we're no longer `adapting to the first image`. 

So I decided to simply add an `overflow: hidden` so the content won't be placed above other sections of the page, in case it overflows.